crates/turborepo-ui/src/tui/app.rs

```diff
@@ -137,18 +306,20 @@ fn run_app_inner<B: Backend + std::io::Write>(
     terminal: &mut Terminal<B>,
     app: &mut App<Box<dyn io::Write + Send>>,
     receiver: AppReceiver,
+    rows: u16,
+    cols: u16,
```

Comment:
Not necessary for this PR, but I think we'll want these bits of info to end up as part of the `App` state. 

It's an implicit part of the `TerminalOutput` and if these values change from what `TerminalOutput` was constructed with, we need to make sure all terminals are resized. Currently these values don't change, but once we tackle handling terminal resizes they will.

Reply:
In realizing I didn't delete any of the unneeded code in `TerminalPane` I ended up getting rid of this. From my rudimentary hand-testing, I'm not seeing any problems but let me know if you find something.